### PR TITLE
docs: release 1.1.0-beta

### PR DESCRIPTION
**Pull request by hetao92**

### CODE REVIEW

File: docs/CHANGELOG-zh.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
 
+## 2020.8.18 - v1.1.0-beta
+- 功能增强:
+  - 图探索 
+    - 增加索引查询节点功能
+    - 增加ID查询预处理功能
+  - 控制台 
+    - 支持节点查询结果导入图探索画板功能
+- 修复
+  -图探索 节点布尔类型属性返回值显示问题修复
```
Comment (Amber1990Zhang) [inline docs/CHANGELOG-zh.md:11]: -图探索 if this is a sub-list, a space is needed here.

File: docs/CHANGELOG-zh.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+- 优化
+  - UI交互优化
```
Comment (Amber1990Zhang) [inline docs/CHANGELOG-zh.md:13]: space between English and Chinese

File: docs/CHANGELOG-zh.md [hunk 3]
```diff
@@ -1,5 +1,18 @@
...
+  - 文案调整
+
 ## 2020.7.31 - v1.0.10-beta
 - 功能增强:
   - 图探索 - 拓展增加limit选项
```
Comment (Amber1990Zhang) [inline docs/CHANGELOG-zh.md:18]: ditto

File: docs/CHANGELOG-zh.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## 2020.8.18 - v1.1.0-beta
+- 功能增强:
+  - 图探索 
+    - 增加索引查询节点功能
+    - 增加ID查询预处理功能
```
Comment (Amber1990Zhang) [inline docs/CHANGELOG-zh.md:7]: space between English and Chinese

File: docs/CHANGELOG-zh.md [hunk 5]
```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## 2020.8.18 - v1.1.0-beta
+- 功能增强:
+  - 图探索 
+    - 增加索引查询节点功能
```
Comment (Amber1990Zhang) [inline docs/CHANGELOG-zh.md:6]: - 增加索引查询节点功能
- 增加ID查询预处理功能
I don’t quite understand here if I don’t try the new features

File: docs/CHANGELOG-en.md [hunk 6]
```diff
@@ -1,5 +1,17 @@
...
 
+## 2020.8.18 - v1.1.0-beta
+- Feature Enhancements:
+  - Explore 
+    - Add index query
+    - Add pre-process vertex id
+  - Console 
+    - Support vertices import graph exploration board
+- Fix:
+  - Explore - Display problem of boolean attributes fix
```
Comment (Amber1990Zhang) [inline docs/CHANGELOG-en.md:11]: Fixed the display problem for boolean properties

File: docs/CHANGELOG-zh.md (hunk 5 above)
Comment (oldLady344) [reply docs/CHANGELOG-zh.md:6]: - 支持使用索引查询点
- 支持先对VID进行预处理，再执行查询？

File: docs/CHANGELOG-zh.md (hunk 4 above)
Comment (oldLady344) [reply docs/CHANGELOG-zh.md:7]: > space between English and Chinese

之前好像都没空格，也可以不需要空格吧。这个文档的样式保持一致就可以了。

File: docs/CHANGELOG-zh.md [hunk 7]
```diff
@@ -1,5 +1,18 @@
...
+    - 增加ID查询预处理功能
+  - 控制台 
+    - 支持节点查询结果导入图探索画板功能
```
Comment (oldLady344) [inline docs/CHANGELOG-zh.md:9]: 支持将点查询结果导入图探索画板

File: docs/CHANGELOG-en.md [hunk 8]
```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 2020.8.18 - v1.1.0-beta
+- Feature Enhancements:
+  - Explore 
+    - Add index query
+    - Add pre-process vertex id
```
Comment (oldLady344) [inline docs/CHANGELOG-en.md:7]: - Support query by index
- Support pre-processing VIDs before query by VID

?

File: docs/CHANGELOG-en.md [hunk 9]
```diff
@@ -1,5 +1,17 @@
...
+  - Console 
+    - Support vertices import graph exploration board
```
Comment (oldLady344) [inline docs/CHANGELOG-en.md:9]: - Support importing the results of vertex query into the graph exploration board

**Review by oldLady344 — APPROVED** [review]

**Pull request merged**